src/model/transactions.js

```diff
@@ -64,6 +64,19 @@ const ErrorDetailsDef = {
 
 // OrchestrationMetadata Schema
 const OrchestrationMetadataDef = {
+  name: {
```

Comment:
Im not sure why we duplicating this schema? What we need to do is assign the `ResponseDefMain` (which will be updated later on) to the `OrchestrationMetadataDef.response`. And the same for the request property. That schema adds the victuals property `body` to the schema

Reply:
Its not being duplicated. Dont know how it came out like this. I added the RoutesOrchestrationsMetaDataDef, which you can see below the comment. I did this to get the tests involving the routes orchestrations to pass. The tests were expecting the old Orchestration schema which has a request and response body. This shall be changed when we work the routes.

Reply:
I dont think its necessary to create the `RoutesOrchestrationMetadataDef` schema. All that we need to is assign the new schema as defined here https://github.com/jembi/openhim-core-js/blob/refactoring-features/src/model/transactions.js#L4-L36 to the orchestration schema definition here https://github.com/jembi/openhim-core-js/blob/refactoring-features/src/model/transactions.js#L71-L74